kbi/0024/index.rst

```diff
@@ -0,0 +1,263 @@
+  │       ├── annex
+  │       │   └── objects
+  │       │       └── Pf
+  │       │           └── vq
+  │       │               └── MD5E-s15--7a593f3460f1efc629489d5a9e86c7b0.txt
+  │       │                   └── MD5E-s15--7a593f3460f1efc629489d5a9e86c7b0.txt
+  │       ├── archives
+  │       │   └── archive.7z
+  │       ├── branches
+  │       ├── config
+  │       ├── config.dataladlock
+  │       ├── description
+  │       ├── HEAD
+  │       ├── objects
+  │       │   ├── 08
+  │       │   │   └── e3ea145c77abd0cf9cb07f04f069efed2bd637
+  │       │   ├── 0d
+  │       │   │   └── 81baa2295544cae79101a18f6473a6c917b927
+  │       │   ├── [...]
+  │       │   │   └── [...]
+  │       │   ├── info
+  │       │   └── pack
+  │       ├── ora-remote-5a413a03-91cb-4433-a2b5-e2d108ec291b
+  │       │   └── transfer
+  │       ├── refs
+  │       │   ├── heads
+  │       │   │   ├── git-annex
+  │       │   │   └── master
+  │       │   └── tags
+  │       └── ria-layout-version
+  ├── error_logs
+  └── ria-layout-version
+
+
+Run the ``cleanup.sh`` script again to update the archive with the new objects::
+
+  $ find my_store -mindepth 2 -maxdepth 2 -type d | xargs -n1 bash cleanup.sh
+  $ tree my_store
+  my_store
+  ├── 1e9
+  │   └── 91f14-baff-4565-8b38-fceed63bb805
+  │       ├── archives
+  │       │   └── archive.7z
+  │       ├── branches
+  │       ├── config
+  │       ├── config.dataladlock
+  │       ├── description
+  │       ├── HEAD
+  │       ├── objects
+  │       │   ├── 08
+  │       │   │   └── e3ea145c77abd0cf9cb07f04f069efed2bd637
+  │       │   ├── 0d
+  │       │   │   └── 81baa2295544cae79101a18f6473a6c917b927
+  │       │   ├── [...]
+  │       │   │   └── [...]
+  │       │   ├── info
+  │       │   └── pack
+  │       ├── refs
+  │       │   ├── heads
+  │       │   │   ├── git-annex
+  │       │   │   └── master
+  │       │   └── tags
+  │       └── ria-layout-version
+  ├── error_logs
+  └── ria-layout-version
+
+
+Let's verify that the archive was updated successfully with the new content,
+using the dataset we started with::
+
+  $ cd my_dataset
+
+This dataset only has one file (``file1.txt``)::
+
+  $ tree
+  .
+  └── file1.txt -> .git/annex/objects/v2/2W/MD5E-s15--af1cdf0b10caa12cf13312f7bb4215df.txt/MD5E-s15--af1cdf0b10caa12cf13312f7bb4215df.txt
+
+
+Run ``datalad update`` to bring in the updates from the RIA store (i.e. ``file2.txt``)::
+
+  $ datalad update --merge
+  $ tree
+  .
+  ├── file1.txt -> .git/annex/objects/v2/2W/MD5E-s15--af1cdf0b10caa12cf13312f7bb4215df.txt/MD5E-s15--af1cdf0b10caa12cf13312f7bb4215df.txt
+  └── file2.txt -> .git/annex/objects/Pf/vq/MD5E-s15--7a593f3460f1efc629489d5a9e86c7b0.txt/MD5E-s15--7a593f3460f1efc629489d5a9e86c7b0.txt
+
+
+
+.. literalinclude:: cleanup.sh
+   :language: bash
+   :linenos:
+   :caption: cleanup.sh
+
+.. _ria stores: http://handbook.datalad.org/en/latest/beyond_basics/101-147-riastores.html
+.. _7-Zip: https://www.7-zip.org/
```

Comment:
```suggestion
.. _7-Zip: https://www.7-zip.org/
.. _inodes: https://en.wikipedia.org/wiki/Inode
```

Related to my first suggestion. I'm fine with rejecting or using a different target.